dashboards/localvolumeprovisioner.yaml

```diff
@@ -0,0 +1,470 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: prometheus-operator-grafana
+    grafana_dashboard: "1"
+  name: prometheus-kubeaddons-prom-local-volume-provisioner
+data:
+  localvolumeprovisioner.json: |-
+    {
+       "__inputs": [ ],
+       "__requires": [ ],
+       "annotations": {
+          "list": [ ]
+       },
+       "editable": false,
+       "gnetId": null,
+       "graphTooltip": 0,
+       "hideControls": false,
+       "id": null,
+       "links": [ ],
+       "refresh": "",
+       "panels": [
+          {
+             "aliasColors": { },
+             "bars": false,
+             "dashLength": 10,
+             "dashes": false,
+             "description": "Discovered volumes per worker",
+             "fill": 1,
+             "gridPos": {
+                "h": 9,
+                "w": 12,
+                "x": 0,
+                "y": 0
+             },
+             "id": 2,
+             "interval": "15s",
+             "legend": {
+                "avg": false,
+                "current": false,
+                "max": false,
+                "min": false,
+                "show": false,
+                "total": false,
+                "values": false
+             },
+             "lines": true,
+             "linewidth": 1,
+             "links": [ ],
+             "nullPointMode": "null",
+             "percentage": false,
+             "pointradius": 2,
+             "points": false,
+             "renderer": "flot",
+             "seriesOverrides": [],
+             "spaceLength": 10,
+             "stack": false,
+             "steppedLine": true,
+             "targets": [
+                {
+                   "expr": "local_volume_provisioner_persistentvolume_discovery_total{instance=~\"$instance\"}",
+                   "format": "time_series",
+                   "instant": false,
+                   "interval": "",
+                   "intervalFactor": 4,
```

Comment:
Based on some experiments I did, it looks like that the 1/1 resolution makes the graph a bit noisier, and the 1/2 resolution seems to deliver almost the same amount of information, especially in the heatmap.

Since the heatmap is now removed, the 1/1 resolution looks not as noisy in the new graphs. I'll go with your suggestion.